.github/workflows/publish.yml

```diff
@@ -64,12 +65,6 @@ jobs:
       - name: Build
         run: pnpm build
 
-      - name: Publish dev packages
-        if: ${{ steps.changesetsCheck.outputs.hasChangesets == 'true' && steps.is_dev.outputs.value == 'true' }}
```

Comment:
The `is_dev` step above can be removed now, I think. Side-note: shouldn't the Build step above set the `NODE_ENV`?

Reply:
yeah i think u correct here